tests/translate/translate_map1_ppm_2d.py

```diff
@@ -50,6 +50,9 @@ def compute(self, inputs):
         inputs["j_2d"] = self.grid.global_to_local_y(
             inputs["j_2d"] + TranslateGrid.fpy_model_index_offset
         )
+        inputs["j1"] = inputs["j_2d"]
+        inputs["j2"] = inputs["j_2d"]
+        del inputs["j_2d"]
```

Comment:
Same as above...

Reply:
This one I'd like to keep just so there's some concept of what j-slice is being tested in the translate class

Reply:
Great!